test/Interop/Cxx/modules/emit-module-interface.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -enable-library-evolution -swift-version 5 -Xfrontend -enable-experimental-cxx-interop %S/Inputs/namespace-extension-lib.swift -emit-module -emit-library -emit-module-interface -emit-module-interface-path %t/UsesCxxStruct.swiftinterface -static -module-name UsesCxxStruct
+// RUN: %FileCheck --input-file=%t/UsesCxxStruct.swiftinterface %s
```

Comment:
It would be interesting to verify that the generated swiftinterface can be parsed. There are macros for that, here's a sample usage:

```
// RUN: %target-swift-emit-module-interface(%t/Original.swiftinterface) %t/Original.swift
// RUN: %target-swift-typecheck-module-from-interface(%t/Original.swiftinterface)
```

Reply:
Thanks, I didn't know about these macros. Added `target-swift-typecheck-module-from-interface` as well.